package.json

```diff
@@ -68,67 +71,70 @@
   },
   "dependencies": {
     "async": "^2.5.0",
-    "autoprefixer": "^8.5.2",
+    "autoprefixer": "^9.5.1",
     "axe-core": "3.0.3",
-    "babel-loader": "^7.1.2",
+    "babel-loader": "^8.0.5",
     "babel-polyfill": "^6.23.0",
-    "browserslist-config-terra": "^1.0.0",
     "chai": "^4.2.0",
     "chalk": "^2.4.2",
-    "clean-webpack-plugin": "^0.1.17",
+    "clean-webpack-plugin": "^2.0.1",
     "clone": "^2.1.1",
     "commander": "^2.19.0",
-    "css-loader": "^0.28.7",
+    "css-loader": "^2.1.1",
+    "cssnano": "^4.1.10",
+    "cssnano-preset-advanced": "^4.0.7",
     "duplicate-package-checker-webpack-plugin": "^3.0.0",
     "express": "^4.16.4",
-    "file-loader": "^1.1.5",
-    "fs-extra": "^6.0.1",
+    "file-loader": "^3.0.1",
+    "fs-extra": "^7.0.1",
     "glob": "^7.1.1",
     "ip": "^1.1.5",
     "load-json-file": "^5.0.0",
     "memory-fs": "^0.4.1",
     "mime-types": "^2.1.21",
-    "mini-css-extract-plugin": "^0.4.5",
+    "mini-css-extract-plugin": "^0.6.0",
     "node-sass": "^4.11.0",
-    "postcss": "^6.0.9",
-    "postcss-assets-webpack-plugin": "^2.0.0",
-    "postcss-custom-properties": "^7.0.0",
-    "postcss-loader": "^2.0.6",
+    "postcss": "^7.0.16",
+    "postcss-assets-webpack-plugin": "^3.0.0",
+    "postcss-custom-properties": "^8.0.10",
+    "postcss-loader": "^3.0.0",
     "postcss-rtl": "^1.3.3",
     "raf": "^3.4.1",
-    "raw-loader": "^0.5.1",
+    "raw-loader": "^2.0.0",
     "sass-loader": "^7.0.1",
     "shelljs": "^0.8.3",
     "terra-aggregate-translations": "^1.0.0",
     "terser-webpack-plugin": "^1.1.0",
     "wdio-visual-regression-service": "^0.9.0",
     "webdriverio": "^4.14.1",
-    "webpack": "^4.28.1",
-    "webpack-cli": "^3.0.0",
-    "webpack-dev-server": "^3.1.11",
+    "wdio-mocha-framework": "^0.5.11",
```

Comment:
wdio-mocha-framework `0.6.4` is available. The caret doesn't work as one might think when the major version is 0. If we want to pull in the latest minor and patch release of wdio-mocha-framework while its major version is 0, we can use `0.x` as the version number. You can test this out on https://semver.npmjs.com/ to see what versions `0.x` will match.

Reply:
I think I totally missed this because this used to be an optional dependency but when I tested removing it all together the tests failed so I moved it to be a direct dependency. I just dug through the commits for changed with this and wdio-sync (it's dependency), this bump actually resolves the errors I was seeing when installing with node 10!

Good catch.